controllers/admin/AdminBlockListingController.php

```diff
@@ -165,9 +168,9 @@ public function displayAjaxSaveBlockContent()
             $blockPsr->custom_icon = '';
         }
         $blockPsr->date_add = date('Y-m-d H:i:s');
-        $blockPsr->date_update = date('Y-m-d H:i:s');
+        $blockPsr->date_upd = date('Y-m-d H:i:s');
```

Comment:
Nice catch! So this PR not only improves the project, it introduces bug fixes too! 👏

Reply:
:+1: